storage/oasis/nodeapi/enigma/convert.go

```diff
@@ -0,0 +1,480 @@
+package enigma
```

Comment:
I'm just gonna throw this in here: `estar` would be a less creative option before the name somehow gets chosen

Reply:
Hehe
:shrug: 
`estoy` indiferente
Not a bad suggestion though.